express-error-handling/server.ts

```diff
@@ -0,0 +1,115 @@
+import express from 'express';
+import { errorMiddleware, ClientError } from './lib/index.js';
+
+type Note = {
+  noteId: number;
+  content: string;
+};
+
+const app = express();
+
+app.get('/api/notes', async (req, res, next) => {
+  try {
+    const notes = await readNotes();
+    res.send(notes);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.post('/api/notes', async (req, res, next) => {
+  try {
+    const { content } = req.query;
+    if (content === undefined) {
+      throw new ClientError(400, 'must have value');
+      return;
```

Comment:
This return is not needed since the `throw` will stop execution. This line of code (25) will never execute. This is true throughout the entire PR.